ml_peg/calcs/molecular/Wiggle150/calc_Wiggle150.py

```diff
@@ -0,0 +1,236 @@
+        """
+        Evaluate potential energy for a structure.
+
+        Parameters
+        ----------
+        atoms
+            Structure to evaluate.
+        calc
+            ASE calculator instance.
+
+        Returns
+        -------
+        float
+            Potential energy in eV.
+        """
+        atoms_copy = atoms.copy()
+        atoms_copy.calc = calc
+        energy = atoms_copy.get_potential_energy()
+        return float(energy)
+
+    @staticmethod
+    def benchmark_wiggle150(
+        calc: Calculator, model_name: str, base_dir: Path
+    ) -> list[Atoms]:
+        """
+        Run Wiggle150 benchmark for a given calculator.
+
+        Parameters
+        ----------
+        calc
+            ASE calculator for predictions.
+        model_name
+            Name of the model.
+        base_dir
+            Path to extracted Wiggle150 data.
+
+        Returns
+        -------
+        list[Atoms]
+            Conformer structures annotated with prediction metadata.
+        """
+        molecules = Wiggle150Benchmark.load_structures(base_dir)
+        conformer_atoms: list[Atoms] = []
+
+        for molecule in MOLECULE_ORDER:
+            entries = molecules[molecule]
+            ground = entries["ground"]
+            ground_label, ground_ref_energy = Wiggle150Benchmark.parse_structure_info(
+                ground
+            )
+            ground_energy_model = Wiggle150Benchmark.get_energy(ground, calc)
+
+            for atoms in tqdm(
+                entries["conformers"], desc=f"Wiggle150 {molecule.upper()}"
+            ):
+                label, ref_energy = Wiggle150Benchmark.parse_structure_info(atoms)
+                model_energy = Wiggle150Benchmark.get_energy(atoms, calc)
+
+                rel_energy_model_kcal = (
+                    model_energy - ground_energy_model
+                ) * EV_TO_KCAL_PER_MOL
+                rel_energy_ref_kcal = ref_energy - ground_ref_energy
+                error_kcal = rel_energy_model_kcal - rel_energy_ref_kcal
+
+                annotated = atoms.copy()
+                annotated.calc = None
+                annotated.info.clear()
+                annotated.info.update(
+                    {
+                        "structure": label,
+                        "molecule": molecule,
+                        "ground_state": ground_label,
+                        "relative_energy_ref_kcal": rel_energy_ref_kcal,
+                        "relative_energy_pred_kcal": rel_energy_model_kcal,
+                        "relative_energy_error_kcal": error_kcal,
+                        "model": model_name,
+                    }
+                )
+                conformer_atoms.append(annotated)
+
+        return conformer_atoms
+
+    def run(self) -> None:
+        """Execute Wiggle150 benchmark calculations."""
+        calc = self.model.get_calculator()
+
+        base_dir = get_benchmark_data(DATA_ARCHIVE) / DATA_SUBDIR
```

Comment:
This branch needs rebasing and this should be swapped out with the one of the new download functions